### PR TITLE
CC-16286 finish guest-cart tests

### PR DESCRIPTION
**Pull request by pyvovarov-s**
CC-16286 finish guest-cart tests

### CODE REVIEW

File: tests/api/b2c/glue/cart_endpoints/guest-carts/positive.robot [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    And Each array element of array in response should contain value:    [included]    amount
+    And Each array element of array in response should contain value:    [included]    code
+    And Each array element of array in response should contain value:    [included]    discountType
+    And Each array element of array in response should contain value:    [included]    displayName
+    And Each array element of array in response should contain value:    [included]    isExclusive
+    And Each array element of array in response should contain value:    [included]    expirationDateTime
+    And Each array element of array in response should contain value:    [included]    discountPromotionAbstractSku
+    And Each array element of array in response should contain value:    [included]    discountPromotionQuantity
+
+Converting_guest_cart_to_regular
```
Comment (OKrivtsova) [inline tests/api/b2c/glue/cart_endpoints/guest-carts/positive.robot:129]: this is not converting, this is merging of the carts

in your case it is merged with the regular cart. but then you need to have an item in regular cart, then have item in guest cart and then they merge and you have regular cart with 2 items.

another case is creating guest cart, then logging in and the item from guest cart is in regular cart.

File: tests/api/b2c/glue/cart_endpoints/guest-carts/positive.robot [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    And Response body parameter should be:    [data][attributes][currency]    ${currency_code_eur}
+    And Response body parameter should be:    [data][attributes][store]    ${store_de}
+    And Response body parameter should not be EMPTY:    [data][attributes][totals][expenseTotal]
+    And Response body parameter should not be EMPTY:    [data][attributes][totals][discountTotal]
+    Response body parameter should be greater than:    [data][attributes][totals][taxTotal]    0
+    Response body parameter should be greater than:    [data][attributes][totals][subtotal]    0
+    Response body parameter should be greater than:    [data][attributes][totals][grandTotal]    0
+    Response body parameter should be greater than:    [data][attributes][totals][priceToPay]    0
+
+Retrieve_guest_cart_including_cart_items
```
[the remark this answers is not in the document]
Comment (pyvovarov-s) [reply tests/api/b2c/glue/cart_endpoints/guest-carts/positive.robot:60]: we will cover them in the items endpoints

File: tests/api/b2c/glue/cart_endpoints/guest-carts/positive.robot (hunk 1 above)
Comment (pyvovarov-s) [reply tests/api/b2c/glue/cart_endpoints/guest-carts/positive.robot:129]: done as was discussed

Comment (OKrivtsova) [reply tests/api/b2c/glue/cart_endpoints/guest-carts/positive.robot]: please add products as discussed and check this
image

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**